master/pkg/schemas/expconf/legacy_test.go

```diff
@@ -214,8 +216,10 @@ func TestLegacyConfig(t *testing.T) {
 				},
 				bindMounts: BindMountsConfig{},
 				envvars: EnvironmentVariablesMap{
-					RawCPU: []string{"HOME=/where/the/heart/is"},
-					RawGPU: []string{"HOME=/where/the/cuda/is"},
+					RawCPU:  []string{"HOME=/where/the/heart/is"},
+					RawCUDA: []string{"HOME=/where/the/cuda/is"},
+					RawGPU:  []string{"HOME=/where/the/cuda/is"},
+					RawROCM: []string{},
```

Comment:
seems like if the legacy experiment config has a `gpu` field, that should populate the modern `CUDA` and `ROCM` fields, no?